addons/zeus/functions/fnc_getModuleDestination.sqf

```diff
@@ -5,7 +5,14 @@
  * Arguments:
  * 0: The souce object <OBJECT>
  * 1: Code to run when position is ready <CODE>
- * - Code is passed [0: Successful <BOOL>, 1: Object <OBJECT>, 2: Position ASL <ARRAY>]
+ * - Code is passed [
+                        0: Successful <BOOL>, 
+                        1: Object <OBJECT>, 
+                        2: Position ASL <ARRAY>, 
+                        3: State of Shift <BOOL>, 
+                        4: State of Ctrl <BOOL>,
+                        5: State of Alt <BOOL>
```

Comment:
I just indented this because it got messy with more than 3 params. This are the params that are passed to the code that is run when the user finished placing. I don't know how you handle such things in your docs